app/Http/Controllers/Admin/Nodes/NodeViewController.php

```diff
@@ -122,7 +123,7 @@ public function allocations(Request $request, Node $node)
             'node' => $node,
             'allocations' => Allocation::query()->where('node_id', $node->id)
                 ->groupBy('ip')
-                ->orderByRaw('INET_ATON(ip) ASC')
+                ->orderByRaw(DB::getDriverName() === 'pgsql' ? 'ip ASC' : 'INET_ATON(ip) ASC')
```

Comment:
As Laravel supports PostgreSQL, out-of-the-box this shouldn't be necessary, as Eloquent and the query builder, knows how to form it.

Reply:
Laravel supporting Postgre doesn't mean that every function of every database engine is implemented in Eloquent.

Reply:
Eloquent and the query builder do not know how to form it, as sergiz has pointed out.